cluster/gke/util.sh

```diff
@@ -181,6 +181,11 @@ function kube-up() {
 
   # Bring up the cluster.
   "${GCLOUD}" ${CMD_GROUP:-} container clusters create "${CLUSTER_NAME}" "${create_args[@]}"
+
+  if [[ ! -z "${HEAPSTER_MACHINE:-}" ]]; then
```

Comment:
I think we should proceed this way, so if you can finish it, it would be awesome.